### PR TITLE
Added option to change loglevel of logger handler on runtime

### PR DESCRIPTION
**Pull request by igal-stratoscale**
Hi,
added option to change log handlers loglevel on the fly, please review.

@rom-stratoscale @michaelk-stratoscale @mickey-stratoscale @romang-stratoscale @sergey-stratoscale

### CODE REVIEW

File: py/strato/common/log/configurelogging.py [hunk 1]
```diff
@@ -72,6 +72,11 @@ def addFileHandler(name, path):
     logging.info("Logging started for %s" % name)
 
 
+def changeHandlerLogLevelbyHandlerType(logger, logLevel, handlerType=None):
+    """ if not specifying handlers type , change log logleve to all """
```
Comment (rom-stratoscale) [inline py/strato/common/log/configurelogging.py]: level

Comment (rom-stratoscale) [inline py/strato/common/log/configurelogging.py]: why do u set handlerType to None? It's not acceptable value.
Just add: assert handlerType is not None and dont run the loop.

Comment (igal-stratoscale) [inline py/strato/common/log/configurelogging.py]: Why it cant be null? I meant that if it is null change level for all handlers

File: py/strato/common/log/configurelogging.py [hunk 2]
```diff
@@ -72,6 +72,11 @@ def addFileHandler(name, path):
...
+    [handler.setLevel(logLevel) for handler in logger.handlers if not handlerType or type(handler) == handlerType]
```
Comment (rom-stratoscale) [inline py/strato/common/log/configurelogging.py]: Does setLevel changes in place?

Comment (igal-stratoscale) [inline py/strato/common/log/configurelogging.py]: yap

File: py/strato/common/log/configurelogging.py (hunk 1 above)
Comment (igal-stratoscale) [inline py/strato/common/log/configurelogging.py]: Why it cant be null? I meant that if it is null change level for all
handlers

File: py/strato/common/log/configurelogging.py (hunk 2 above)
Comment (igal-stratoscale) [inline py/strato/common/log/configurelogging.py]: yap

File: py/strato/common/log/configurelogging.py (hunk 1 above)
Comment (rom-stratoscale) [inline py/strato/common/log/configurelogging.py]: Ohh. Missed it. So please make the variable more descriptive. None means nothing, not everything. It's not clear enough

**Pull request merged**